### PR TITLE
add maximum text length option

### PR DESCRIPTION
**Pull request by otacke**
Sorry. I screwed up some git stuff. This is the same request as pull request https://github.com/h5p/h5p-text-input-field/pull/5

A teacher may require to limit the number of characters for an input field, e.g. for an essay. This pull request adds the appropriate optional setting to the editor. If a number is given, a dynamic message will display the number of characters that the student has left below the input field. The message can be altered by the teacher including a variable for the number of characters left.

### CODE REVIEW

**Comment by otacke** [comment]
Please note that this pull request will conflict with https://github.com/h5p/h5p-text-input-field/pull/6 If you want to merge with both, I can provide you with a new pull request from my [essay_mix branch](https://github.com/otacke/h5p-text-input-field/tree/essay_mix) where both are combined.

**Comment by timothyylim** [comment]
@otacke , the new fields have to be added for all the languages. Frode has made a script to make that relatively easy to do. Upgrade to the latest version of the CLI:

It's based on the `add-english-texts` command the difference is that it updates all the language files at once using semantics.json and support a special property `state`. E.g. if you add new field in the middle of the semantics you'll want to add `"state": "new"` and it will be inserted into all the translation files. Also supported are: "updated" (resets the translations for field) and "removed"

File: text-input-field.js [hunk 1]
```diff
@@ -30,6 +31,10 @@ H5P.TextInputField = (function ($) {
       requiredField: false
     }, params);
 
+    // Set the maximum length for the textarea
+    this.maxTextLength = (typeof this.params.maximumLength === 'undefined') ?
```
Comment (timothyylim) [inline text-input-field.js]: Better to put this all one big line.

Comment (otacke) [reply text-input-field.js]: Sorry. I had not noticed that you included all the new language definitions after I created the pull request. Back then, there were only three + English. I'll add the new ones today.

**Pull request merged**